graphistry/embed_utils.py

```diff
@@ -412,18 +412,33 @@ def predict_links(
         if source is None:
             src = pd.Series(all_nodes)
         else:
+            # this is temporary
+            try:
```

Comment:
maybe clearer what's happening:
*  `if not isinstance(source, pd.DataFrame): ...` 
* a comment of why/when to get rid of this